### ISSUE TITLE
(Epic) Add xml comments for the whole library

### ISSUE THREAD

**Issue by anranruye** [ant-design-blazor/ant-design-blazor#1495]
### Is your feature request related to a problem? 

Without xml comments,  it's hard to get enough information from the library metadata.

**Comment by anddrzejb** [comment]
We should at least try to add to every `[Parameter]` & public method.

**Comment by knight1219** [comment]
Would we want to pull the documentation from what is in the API section of the current docs for each property?

![image](https://user-images.githubusercontent.com/6771697/137592543-0b647140-6ba7-4149-9a22-b6f583920db8.png)


It would be interesting if once the XML docs are updated that API section could be generated _from_ the XML docs so they stay in sync with each other.

**Comment by ElderJames** [comment]
@knight1219  Yes, we wish we could do that. I would also like to support English and Chinese. Do you have any ideas, please?

**Comment by anddrzejb** [comment]
Cool! But we are going to loose all the xml attributes (like `<see cref=... `), aren't we?

**Comment by knight1219** [comment]
Depends on what you mean: if you mean references to objects, I don't think so. Those should still work. If you mean links to the docs site, we will need to do some find and replace magic to get the correct language in the route

**Comment by knight1219** [comment]
It seems that this isn't a built in way to do localized comments. The roslyn team has had this as an open issue since 2015 with no real signs of being adopted by them. [https://github.com/dotnet/roslyn/issues/3371](Source:dotnet/roslyn). 

I'm still trying to think of the best way to do this so we can support multiple languages, but this is what I've found so far.

**Comment by knight1219** [comment]
Wanted to give an update on this. I have found a [library ](https://github.com/devlooped/NuDoq) that allows your to read and manipulate the XML document file and I am testing it out. The idea is that inside the documentation sections your will add a new `<localize>` node in the XML with the appropriate `lang` attribute:

```
        /// <summary>
        /// <localize lang="en-US">
        /// Offset from the bottom of the viewport (in pixels)
        /// </localize>
        /// <localize lang="zh-CN">
        /// 从视口底部偏移（以像素为单位）
        /// </localize>
        /// </summary>
```

We can then use the NuDoc library to parse out and create as many doc files as we need for the different languages that are part of the documentation. I'm still working on this and hope to have something testable soon. Seeing as this is .net code I'm running, how do we want to get this into the build process? Just a console app that runs and generates everything? I'm open to ideas.

**Comment by knight1219** [comment]
@ElderJames @anddrzejb Does the AntDesign.Docs.Build.CLI project get used in the build process? If so, I could add a new command there and generate the XML files from there.

**Comment by ElderJames** [comment]
@knight1219 Yes, we use the `AntDesign.Docs.Build.CLI` project in the `AntDesign.Docs.Build` project.

**Comment by ElderJames** [comment]
We can refer this sample to localize our package.
https://github.com/NuGet/Samples/tree/main/Satellite-Packages

**Comment by ElderJames** [comment]
We have achieve the localized intellisense via https://github.com/stratosblue/IntelliSenseLocalizer.
